README.md

```diff
@@ -105,7 +105,7 @@ However, when working with larger datasets, it is recommended to perform data pr
 The following command-line will prepare the data and cache it, then use it to train a model.
 ```bash
 # First prepare the data and cache it in `path_to_cached_data`
-graphium-prepare-data datamodule.args.processed_graph_data_path=[path_to_cached_data]
+graphium data prepare ++datamodule.args.processed_graph_data_path=[path_to_cached_data]
```

Comment:
`graphium-train` is the hydra endpoint - specifically meant for training and finetuning. `graphium` is the more general CLI endpoint, organized with various sub commands. Ideally I would've also made the training and finetuning accessible under the `graphium` command, for example under the `graphium train` subcommand, but as I tried to explain in https://github.com/datamol-io/graphium/pull/432#issuecomment-1690082766, the hydra CLI cannot be a subcommand of another CLI, which is why we had to [define](https://github.com/datamol-io/graphium/blob/main/pyproject.toml#L65-L68) a second command. Since the command cannot have any spaces, I think this is a reasonable name for it.

Reply:
Okok I see